### DISCUSSION TITLE
Dark mode / light mode

### DISCUSSION THREAD

**Discussion by EliteAMDGamer** [luisbocanegra/kde-material-you-colors#230] [Q&A] [answered]
Is there a way to toggle the dark or light mode or am I missing something? I've read the FAQ and checked the other discussions or issues. 

The last question I have would it be possible in a future version to have the "View"  Alternative Background color synced with normal background so dolphin doesn't have that alternating light to dark color on details view. As it is now i have to match it manually. 

![image](https://github.com/user-attachments/assets/cdd294a8-4209-4075-a2a0-9e02269911f0)
![image](https://github.com/user-attachments/assets/6d0b73df-2f44-49de-b06b-18aa2d858dec) Stock
![image](https://github.com/user-attachments/assets/5d40f0dd-3194-4f32-abae-b532d118f023) Fixed

**Comment by luisbocanegra** [comment, marked as answer]
> Is there a way to toggle the dark or light mode or am I missing something? I've read the FAQ and checked the other discussions or issues.

Yes, multiple ways

- Running `kde-material-you-colors -d` readme doesn't mention it but tells you how to get the available options using `kde-material-you-colors -h` https://github.com/luisbocanegra/kde-material-you-colors?tab=readme-ov-file#from-terminal
- From the configuration file https://github.com/luisbocanegra/kde-material-you-colors?tab=readme-ov-file#configuration-file
  ```
  # Enable Light mode
  # Accepted values are True or False
  # Commented by default to follow System Color Setting (Material You Light/Dark only)
  # NOTE:
  # Will fallback to dark mode if not defined here or enabled in Settings
  #light = False
  ```
- And the easiest, directly from the widget
  
  ![Screenshot_20241021_105442](https://github.com/user-attachments/assets/d3560f97-bcb5-40e7-b086-659f41c250ad)

> The last question I have would it be possible in a future version to have the "View" Alternative Background color synced with normal background so dolphin doesn't have that alternating light to dark color on details view. As it is now i have to match it manually.

Hmm, personally I prefer lists having an alternate color they make it easier to read long lists with short rows. Instead of adding this specific option it would be better to have something like a template where you define what color should each element take.

**Reply by EliteAMDGamer** [reply]
That's perfect it answers everything for me. I hope one day kde allows me to change that color setting only for dolphin. Other areas when its thin and contrasting I don't get a migraine scrolling. I know I can make it smaller but it doesn't look as nice.
